.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version:
+          - '2.1.0'
+          - '2.3.8'
+          - '2.4.5'
+          - '2.5.3'
+          - '2.6.3'
+          - 'head'
+        gem: ['sunspot', 'sunspot_rails', 'sunspot_solr']
+        update-format: ['xml', 'json']
+        exclude:
+          - ruby-version: 'head'
+            gem: 'sunspot_solr'
+          - ruby-version: 'head'
+            update-format: 'json'
+        include:
+          - ruby-version: '2.6'
+            gem: 'sunspot_solr'
+            update-format: ''
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true # 'bundle install' and cache are handled automatically
+    - name: Uninstall and Install Bundler
```

Comment:
I think latest `setup-ruby` action allows explicit bundler versions out of the box https://github.com/ruby/setup-ruby/blob/master/action.yml#L19